content/posts/2023/reana-0.9.0.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "REANA 0.9.0 is released"
+date: 2023-01-24T09:00:00+01:00
+---
+
+We are happy to announce REANA 0.9.0, a major update that allows
+users to set custom workflow retention rules to automatically delete workspace
+files, adds a new way of launching workflows from external sources, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### Automatically delete workspace files using custom retention rules
+
+It's helpful to have a way to automatically delete files in your workflow's
+workspace that you don't need anymore. For example, if your workflow creates
+large temporary files that take up a lot of space that consume your
+[disk quota](https://docs.reana.io/advanced-usage/user-quotas), you may want to
+get rid of them after a certain period of time. Therefore, we're introducing the
+ability to set up custom rules for keeping or deleting files in your workspace.
+This can save you from having to manually delete them, which can be a
+time-consuming process.
+
+![Retention rules in the web UI](/images/ui-retention-rules.png)
+
+Please see the dedicated blog post about [custom file retention rules](/posts/2022/workspace-file-retention-rules/)
+to learn more about this feature.
+
+### Launching workflows from external sources using web interface
+
+You can now easily run your analysis workflows from publicly accessible external
+sources, like GitHub, GitLab, digital repositories like Zenodo or any website on
+the web using the new REANA launcher page. You can host your workflows on a
+public location and be able to access and run them on REANA without having to do
+any manual steps.
+
+[![Launch home](/images/reana-0.9.0-launcher.png)](https://reana.cern.ch/launch)
+
+For more information about this new feature please check the dedicated blog post
+about [launching workflows from external sources using web interface](/posts/2022/launching-workflows/).
+
+### Rucio authentication for workflow jobs
+
+[Rucio](https://rucio.cern.ch/) is a scientific data management system used in
+LHC particle physics and related scientific domains. It provides access for
+large volumes of data spread across facilities at multiple institutions.
+
+If your workflow needs to use data that is managed by a Rucio system, you can
+follow the documentation that describes the [Rucio authentication method](https://docs.reana.io/advanced-usage/access-control/rucio/)
+to learn more about it.
```

Comment:
It would be useful to print here a "screenshot" of the say Serial example or the Yadage example, so that people can more easily relate to what this brings.

(And we'll have a nice balance that each major feature is  accompanied by a paragraph and an illustrating screenshot.)

```
rule mystep:
  container:
    "docker://reana-env-rucioclient:1.0"
  resources:
    voms_proxy=True,
    rucio=True
  shell:
    "rucio get my_rucio_scope:my_rucio_file"
```

You can get the source for the example with highlighting of given lines from https://docs.reana.io/advanced-usage/access-control/rucio/